flink-streaming-java/src/main/java/org/apache/flink/streaming/api/functions/source/ContinuousFileReaderOperator.java

```diff
@@ -340,6 +341,10 @@ public void run() {
 					}
 				}
 
+			} catch (FileNotFoundException e) {
+				if (LOG.isDebugEnabled()) {
```

Comment:
This needs to be more prominently logged than `debug`. Should be at least `info` or `warn`.

Please also use the placeholder syntax from log4j, to make the code simpler.